docs/design/config-api.md

```diff
@@ -1,17 +1,6 @@
 Configuration API Design
 ===================
 
-# Table of Contents
-1. [Motivation](#motivation)
-2. [Goals](#goals)
-3. [Non-Goals](#non-goals)
-4. [Control Plane API](#control_plane_api)
-   1. [Gateway Type](#gateway)
-   2. [Provider Type](#provider)
-   3. [Configuration Examples](#control_plane_configuration)
-6. [Data Plane API](#data_plane_api)
-   1. [Configuration Examples](#data_plane_configuration)
-
```

Comment:
Why remove the TOC?

Reply:
This will be shown as the title of this post